pkg/app/controller/controller.go

```diff
@@ -0,0 +1,12 @@
+package controller
+
+import (
+	"github.com/aserto-dev/aserto-management/controller"
+	"github.com/aserto-dev/go-aserto/client"
+
+	"github.com/rs/zerolog"
+)
+
+func NewControllerFactory(logger *zerolog.Logger, cfg *controller.Config, dop client.DialOptionsProvider) *controller.Factory {
```

Comment:
Why do we need this level of indirection? This function is basically an alias for `controller.NewFactory`.